netbox/netbox/views/generic/object_views.py

```diff
@@ -362,6 +362,13 @@ def get(self, request, *args, **kwargs):
             **self.get_extra_context(request, obj),
         })
 
+    def save_related_data(self, request, form, objd):
```

Comment:
I like the approach, however we can't add new methods to generic views arbitrarily as they're part of the [plugins API](https://docs.netbox.dev/en/stable/plugins/development/views/), and adding a new method could conflict with a custom method defined by a plugin. It might be worth starting a FR to introduce this method in a future release, but for now it's probably sufficient to implement a one-off just for #10719.